code/modules/mod/mod_control.dm

```diff
@@ -216,6 +216,7 @@
 			. += span_notice("You could remove [ai] with an <b>intellicard</b>.")
 		else
 			. += span_notice("You could install an AI with an <b>intellicard</b>.")
+	. += span_notice("<i>You could examine it more thoroughly...</i>")
```

Comment:
This feels like something that should be automated, but i guess that's for another PR